src/styles/index.scss

```diff
@@ -1,3 +1,62 @@
+@import './utils/variable';
+
 body {
   margin: 0;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  font-size: $main-font-size;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.calendar {
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+  gap: $main-gap;
+
+  width: $main-width;
+  padding: $padding-calendar;
+
+  @each $day, $value in $days-of-week {
+    &--start-day-#{$day} > :first-child {
+      margin-left: ($width-days * $value) + $value;
+    }
+  }
+
+  &__day {
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    width: $width-days;
+    height: $hight-days;
```

Comment:
There seems to be a typo in the variable name `$hight-days`. It should be `$height-days` to match the naming convention and to be clear that it refers to 'height'.